src/models/propulsion/FGTurbine.cpp

```diff
@@ -434,7 +440,7 @@ bool FGTurbine::Load(FGFDMExec* exec, Element *el)
     string name = function_element->GetAttributeValue("name");
     if (name == "IdleThrust" || name == "MilThrust" || name == "AugThrust"
         || name == "Injection" || name == "N1SpoolUp" || name == "N1SpoolDown"
-        || name == "N2SpoolUp" || name == "N2SpoolDown")
+        || name == "N2SpoolUp" || name == "N2SpoolDown" || name == "TSFC" || name == "ATSFC")
```

Comment:
Don't use "pre-functions". Instead re-use the existing `tsfc` and `atsfc` attributes (lines 455-458 of `FGTurbine.cpp`)
```c++
  if (el->FindElement("tsfc")) {
    string value = el->GetDataLine();
    if (is_number(value))
      TSFC = new FGRealValue(atof(value.c_str()));
    else
      TSFC = new FGFunction(FDMExec, el);
  }
  if (el->FindElement("atsfc")) {
    string value = el->GetDataLine();
    if (is_number(value))
      ATSFC = new FGRealValue(atof(value.c_str()));
    else
      ATSFC = new FGFunction(FDMExec, el);
  }
```